test/system/700-play.bats

```diff
@@ -196,6 +196,30 @@ EOF
     run_podman rm -a
 }
 
+@test "podman kube play read-only" {
+    YAML=$PODMAN_TMPDIR/test.yml
+    run_podman create --pod new:pod1 --name test1 $IMAGE touch /test
+    run_podman create --pod pod1 --read-only --name test2 $IMAGE touch /test
+    run_podman create --pod pod1 --read-only --name test3 $IMAGE touch /tmp/test
+    run_podman kube generate pod1 -f $YAML
+
+    run_podman pod rm -a
+    run_podman rm -a
+
+    cat $YAML
+    run_podman kube play $YAML
+    run_podman inspect --format "{{.State.ExitCode}}" pod1-test1
+    is "$output" "0" "File system should be read/write"
+    run_podman inspect --format "{{.State.ExitCode}}" pod1-test2
+    is "$output" "1" "File system should be read/only"
+    run_podman inspect --format "{{.State.ExitCode}}" pod1-test3
+    is "$output" "0" "File system should be read/write"
```

Comment:
I think we can squash the three inspects into one querying all containers at once.

Reply:
Or use `podman wait`, but I lean slightly toward the way it is: it's nice to have an exact message for what happened (although the messages could be improved).

I added `run_podman logs` just before this, to investigate the problem on my laptop, and see:
```
142d0f92a92c touch: /tmp/test: Permission denied
```
This causes the `pod1-test3` test to fail. It's not an AVC. I am stuck.